lib/common/asar.js

```diff
@@ -11,55 +11,38 @@
   const cachedArchives = {}
 
   const envNoAsar = process.env.ELECTRON_NO_ASAR && process.type !== 'browser' && process.type !== 'renderer'
-  const isAsarDisabled = function () {
-    return process.noAsar || envNoAsar
-  }
+  const isAsarDisabled = () => process.noAsar || envNoAsar
 
-  const getOrCreateArchive = function (p) {
+  const getOrCreateArchive = p => {
```

Comment:
Can we get a more descriptive name than `p` 😄

Reply:
i'd say yes except that `path` is taken by the module, which is why this was done originally; given that the two calling functions both directly mention paths i think a viewer would be able to understand what it's reffing?